CMakeLists.txt

```diff
@@ -143,7 +106,7 @@ target_include_directories(ccls SYSTEM PRIVATE
 
 install(TARGETS ccls RUNTIME DESTINATION bin)
 
-if(NOT SYSTEM_CLANG AND NOT ${CMAKE_SYSTEM_NAME} STREQUAL Windows)
+if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL Windows)
```

Comment:
I'm pretty sure this entire if can be removed

Reply:
Thanks. Removed